docs/content/primary-key-table/changelog-producer.md

```diff
@@ -26,7 +26,7 @@ under the License.
 
 # Changelog Producer
 
-Streaming queries will continuously produce the latest changes.
+Streaming queries will continuously consume the latest changes.
```

Comment:
Maybe it should be `Streaming Write can continuously produce the latest changes.`?

Reply:
Streaming write can continuously produce the latest changes for streaming read.